sysid-projects/analysis-test/src/main/include/Drivetrain.h

```diff
@@ -4,14 +4,16 @@
 
 #pragma once
 
+#include <fmt/format.h>
 #include <frc/AnalogGyro.h>
 #include <frc/Encoder.h>
-#include <frc/PWMVictorSPX.h>
-#include <frc/SpeedControllerGroup.h>
 #include <frc/controller/PIDController.h>
 #include <frc/controller/SimpleMotorFeedforward.h>
+#include <frc/fmt/Eigen.h>
```

Comment:
What uses `frc/fmt/Eigen.h`?

Reply:
I think that was from us troubleshooting the angular drivetrain test